ebos/equil/initstateequil.hh

```diff
@@ -1852,6 +1852,12 @@ private:
                                             ptable, psat);
             }
         }
+        comm.sum(regionIsEmpty,rec.size());
+        for (size_t r = 0; r < rec.size(); ++r) {
+            if (regionIsEmpty[r] > 0 )
+                Opm::OpmLog::warning("Equilibration region " + std::to_string(r + 1)
+                                     + " has no active cells");
```

Comment:
Does this mean that every MPI process will emit all these messages now or is `OpmLog::warning()` aware of IO ranks?

Reply:
Another thing: The above will emit a diagnostic if a region is empty on at least one process.  Might it make sense to emit the diagnostic only if the region is empty on **all** processes?

Reply:
Yes and no.
Yes the message is printed on all processes. We might want to use  `if(comm.rank() == 0)`before the for-loop.
No, it will only emit the message if all processes have an empty region (note the `comm.sum(...)` above).

Reply:
> it will only emit the message if all processes have an empty region (note the `comm.sum(...)` above).

Then I'm missing something.  Suppose region 3 is empty on rank (process) 4 (i.e., `regionIsEmpty[3] = 1` on rank 4, but `= 0` on all other ranks).  Won't the `comm.sum()` then ensure that `regionIsEmpty[3] = 1` on all processes/ranks?

Reply:
I think @bska is right. It should be `if(regionIsEmpty[r] == numRegions) `

Reply:
> It should be `if(regionIsEmpty[r] == numRegions)`

Or possibly
```C++
if (regionIsEmpty[r] == comm.size())
```
instead.

Reply:
Indeed, you guys are right. I just assumed the value of the variable and its name would correlate.

Nitpicking:
Hence `regionIsEmpty` actually does `numProcsRegionIsEmpty`.
Maybe rename or change the computation to resemble what the name seems to indicate
```
comm.min(regionIsEmpty.data(), regionIsEmpty.size());
...
if(regionIsEmpty[r])
  ...
```

Reply:
> Maybe rename or change the computation to resemble what the name seems to indicate

That's a good idea.  I agree that `comm.min()` is the most suitable reduction here.